yolort/models/transform.py

```diff
@@ -271,6 +273,14 @@ def _resize_image_and_masks(
 
     image = F.interpolate(image[None], size=size, scale_factor=scale_factor, mode='bilinear',
                           recompute_scale_factor=recompute_scale_factor, align_corners=False)[0]
+    if use_square_box:
```

Comment:
Actually the `use_square_box` you are proposing will play the same role as the `fixed_size` in

https://github.com/zhiqwang/yolov5-rt-stack/blob/b0af4a1b17805543f415df705deb66f398b10170/yolort/models/transform.py#L241-L250